### PR TITLE
Generic binaries for running golem from source

### PR DESCRIPTION
**Pull request by maaktweluit**
Part of solving multi-os test scenarios

### CODE REVIEW

File: scripts/test_bin/windows/golem_source_update.bat [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
+call ".\golem_source__prep.bat"
+
+echo "Intstall requirements"
```
Comment (mfranciszkiewicz) [inline scripts/test_bin/windows/golem_source_update.bat]: Typo :)

**Review by ederenn — APPROVED** [review]
scripts are working for me

**Review by mfranciszkiewicz — COMMENTED** [review]
Maybe move the `PROJECT_DIR`, the venv directory and source directory to a separate script? That way we could configure the environment.

**Comment by maaktweluit** [comment]
@mfranciszkiewicz fixed the config split in the mac version, please have a look!

Will apply to linux and mac when reviewed

File: scripts/test_bin/mac/golem_source__prep.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+for CONFIG in ${_CONFIG_OPTIONS}; do
+	echo "Checking $CONFIG"
+	if [ -f $CONFIG ]; then
+		echo EXISTS
+		. $CONFIG
+		break
+	fi
+	echo "Not there"
+done
```
Comment (mfranciszkiewicz) [inline scripts/test_bin/mac/golem_source__prep.sh]: This part could be moved to a separate script

**Review by mfranciszkiewicz — COMMENTED** [review]
Linux and macOS can be merged

File: scripts/test_bin/mac/golem_source__prep.sh (hunk 2 above)
Comment (maaktweluit) [reply scripts/test_bin/mac/golem_source__prep.sh]: Done!

**Comment by maaktweluit** [comment]
@mfranciszkiewicz Thanks for the tips, applied them all! Please check one final time, then we can merge :)

**Review by mfranciszkiewicz — APPROVED** [review]

**Pull request merged**